src/components/DynamicFormContainer/DynamicForm/CustomTemplates/CustomFileWidget.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { FunctionComponent, useCallback, useState } from "react";
+import { useDropzone } from "react-dropzone";
+import { showInKB } from "../../../../common/utils";
+import { Button } from "../../../../UI/Button";
+import { SvgIcon, SvgIconPaperClip, SvgIconX } from "../../../../UI/SvgIcon";
+
+interface FileUploadType {
+  dataURL: string;
+  name: string;
+  size: number;
+  type: string;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const CustomFileWidget: FunctionComponent<any> = ({
+  onChange,
+  multiple,
+  options,
+  disabled,
+}) => {
+  const [uploadedFiles, setUploadedFiles] = useState<FileUploadType[]>([]);
+  const [fileSizeError, setFileSizeError] = useState(false);
+
+  const maxSize = 20000000;
```

Comment:
Can move this out to the top and name it `const MAX_FILE_SIZE = 20000000 // 20mb` so people know the size? Also I think the number is not exactly 20mb..

Reply:
ok